plugins/aiops/pkg/gateway/modeltraining.go

```diff
@@ -36,6 +36,10 @@ func (p *AIOpsPlugin) TrainModel(ctx context.Context, in *modeltraining.ModelTra
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Failed to train model: %v", err)
 	}
+	_, err = p.PutModelTrainingStatus(ctx, &modeltraining.ModelTrainingStatistics{RemainingTime: 3600})
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Failed to put model training status: %v", err)
```

Comment:
Set the code to `Unavailable` which indicates the call can be retried (if the server is temporarily down for example)

Reply:
sorry I put this comment on the wrong line, it should be on line 37

Reply:
Done